### ISSUE TITLE
Support odd version strings

### ISSUE THREAD

**Issue by kfrancis** [jpobst/Prototype.Android.MavenBindings#6]
Hey,

I'm trying to use this with this: https://central.sonatype.com/artifact/org.jboss/jboss-vfs/3.2.17.Final but the ".Final" is causing the following:

```
The "MavenDependencyVerifierTask" task failed unexpectedly.
System.ArgumentException: '3.2.17.Final' is not a valid version string.
Parameter name: value
   at NuGet.Versioning.NuGetVersion.Parse(String value)
   at MavenNet.MavenVersionRange.Satisfies(String version)
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at Prototype.Android.MavenBinding.Tasks.DependencyResolver.IsDependencySatisfied(Dependency dependency, MicrosoftNuGetPackageFinder packages, LogWrapper log)
   at Prototype.Android.MavenBinding.Tasks.MavenDependencyVerifierTask.Execute()
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	Tesseract.Binding.Droid	C:\Users\kfrancis\.nuget\packages\xamprototype.android.mavenbinding.tasks\0.0.7\build\XamPrototype.Android.MavenBinding.Tasks.targets
```

They seem to include that in all their versions: https://central.sonatype.com/artifact/org.jboss/jboss-vfs/3.2.17.Final/versions

**Comment by jpobst** [comment]
Fun!  🙄 

I'll look into what we can do about it.

**Comment by jpobst** [comment]
Released version `0.0.8` which should fix this.
